src/Http/Controllers/TeamspeakController.php

```diff
@@ -393,7 +414,12 @@ public function getUserID() {
     }
 
     public function getRegisterUser() {
-        return view('teamspeak::register');
+        $character = CharacterSheet::find(setting('main_character_id'));
```

Comment:
Same thing as bellow. You must handle the case where the character sheet does not exists (anymore?)